products/customer_analytics/frontend/scenes/CustomerAnalyticsConfigurationScene/events/CustomerAnalyticsDashboardEvents.tsx

```diff
@@ -19,50 +20,67 @@ export interface EventSelectorProps {
     filters: FilterType | null
     setFilters: (filters: FilterType) => void
     title: string
+    prompt: string
 }
 
-function EventSelector({ filters, setFilters, title, caption }: EventSelectorProps): JSX.Element {
+function EventSelector({ filters, setFilters, title, caption, prompt }: EventSelectorProps): JSX.Element {
+    const { openSidePanel } = useActions(sidePanelStateLogic)
     const { eventsToHighlight } = useValues(customerAnalyticsDashboardEventsLogic)
     const highlight = eventsToHighlight.includes(title) ? 'border rounded border-dashed border-danger' : ''
 
     return (
         <div className={`py-2 ${highlight}`}>
             <div className="ml-1">
-                <LemonLabel>{title}</LemonLabel>
+                <div className="flex items-center gap-2">
+                    <LemonLabel>{title}</LemonLabel>
+                    <LemonButton
+                        size="small"
+                        type="tertiary"
+                        icon={<IconSparkles className="text-accent" />}
+                        tooltip="Configure with PostHog AI"
+                        onClick={() => openSidePanel(SidePanelTab.Max, prompt)}
```

Comment:
you could directly use `askMax` to trigger the conversation, without having the user click on submit